File: server/src/models/CartModel.js
const mongoose = require("mongoose");

const schema = mongoose.Schema

const cartSchema = new schema({
    user_id: { type: mongoose.Types.ObjectId,ref:'user_tb' },
    product_name: { type: String },
    product_id: { type: String },
    quantity: { type: String },
    product_image: { type: String },
    price: { type: String },

    
    
})

const cartModel = mongoose.model('cart_tb',cartSchema)
module.exports = cartModel

File: lib/Admin/model/cartModel.dart
class CartModel {
  final String productname;
 // final String category;
  final String quantity;
  final String image;
  final String price;
  final String id;

  CartModel({
    required this.productname,
  //  required this.category,
    required this.quantity,
    required this.image,
    required this.price,
    required this.id,
  });

  factory CartModel.fromJson(Map<String, dynamic> json) {
    return CartModel(
      productname: json['product_name'],
   //   category: json['category'],
      quantity: json['quantity'],
      image: json['product_image'],
      price: json['price'],
      id: json['_id'],
    );
  }
}
